integreat_cms/core/management/commands/update_link_text.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+import logging
+import time
+from html import unescape
+from typing import TYPE_CHECKING
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import CommandError
+from linkcheck.listeners import tasks_queue
+from linkcheck.models import Url
+from lxml.etree import ParserError
+from lxml.html import fromstring, tostring
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.core.management.base import CommandParser
+
+    from .users.user import User
+
+
+from ..log_command import LogCommand
+
+logger = logging.getLogger(__name__)
+
+
+def get_url(url: str) -> Url:
+    """
+    Get an URL object by url or raise an error if not found
+
+    :param url: url as string
+    :return: URL
+    """
+    try:
+        return Url.objects.get(url=url)
+    except Url.DoesNotExist as e:
+        raise CommandError(f'URL object with url "{url}" does not exist.') from e
+
+
+def get_user(username: str) -> User:
+    """
+    Get a user by username or raise an error if not found
+
+    :param username: Username
+    :return: User
+    """
+    try:
+        return get_user_model().objects.get(username=username)
+    except get_user_model().DoesNotExist as e:
+        raise CommandError(f'User with username "{username}" does not exist.') from e
+
+
+class Command(LogCommand):
+    """
+    Management command to update the link text of ALL links with the given URL.
+    ALL links receive the SAME new link text.
+    This applies in ALL regions.
+    """
+
+    help = "Update link text of the URL"
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        """
+        Define the arguments of this command
+
+        :param parser: The argument parser
+        """
+        parser.add_argument(
+            "--target-url", help="The URL whose link text needs to be changed"
+        )
+        parser.add_argument("--new-link-text", help="The new link text")
```

Comment:
If we mark the arguments as [required](https://docs.python.org/3/library/argparse.html#required), they are handled and rendered automatically
```suggestion
        parser.add_argument(
            "--target-url", required=True, help="The URL whose link text needs to be changed"
        )
        parser.add_argument("--new-link-text", required=True, help="The new link text")
```